internal/isoeditor/rhcos_test.go

```diff
@@ -135,12 +141,19 @@ var _ = Context("with test files", func() {
 					scriptBytes, err := ioutil.ReadAll(r)
 					Expect(err).ToNot(HaveOccurred())
 					scriptContent = string(scriptBytes)
+				case "/etc/systemd/system/coreos-livepxe-rootfs.service.d/10-proxy.conf":
+					rootfsServiceConfigBytes, err := ioutil.ReadAll(r)
+					Expect(err).ToNot(HaveOccurred())
+					rootfsServiceConfigContent = string(rootfsServiceConfigBytes)
 				}
 			}
 
 			Expect(configContent).To(Equal("staticipconfig"))
 			Expect(scriptContent).To(Equal(constants.ConfigStaticIpsScript))
 
+			rootfsServiceConfig, _ := editor.(*rhcosEditor).formatRootfsServiceConfigFile(&clusterProxyInfo)
+			Expect(rootfsServiceConfigContent).To(Equal(rootfsServiceConfig))
```

Comment:
I don't think this could ever fail ... We're asserting that the output of `formatRootfsServiceConfigFile` will be the same as the output of `formatRootfsServiceConfigFile`.

We should have the expected text in this test and assert that the file in the archive matches that content rather than calling this private function again here.

Reply:
right, added as a string.